config/locales/devise/devise-en.yml

```diff
@@ -101,7 +111,7 @@ en:
         sign_up_new_account: Sign up for a new account
     sign_in: Sign In
     sign_up: Sign Up
-    user_name: User Name
+    user_name: Username
```

Comment:
I noticed inconsistencies in the labels and error messages for username. Let me know if we should follow User Name or Username. Still we have to make this uniform I believe.